build.gradle

```diff
@@ -66,7 +66,7 @@ dependencies {
     modImplementation include('me.lucko:fabric-permissions-api:0.1-SNAPSHOT')
 
     // Translations
-    modImplementation include("fr.catcore:server-translations-api:${project.server_translations_version}")
+    modImplementation include("com.github.QuiltServerTools:Server-Translations:1.17-SNAPSHOT")
```

Comment:
Convention dictates it shouldn't be capitalised, but I'm not sure if you can change that now